src/flash.t.sol

```diff
@@ -415,93 +415,103 @@ contract DssFlashTest is DSTest {
         assertEq(vat.sin(address(flash)), 0);
     }
 
+    function testFail_flash_vat_not_live() public {
+        vat.cage();
```

Comment:
This is definitely one of those places where foundry's expect fail checks will help, if the test reverts on these vat calls the tests still pass

Reply:
Yep, we can still do with dapptools and those functions I created for `ds-test`.